modules/api/src/main/java/org/apache/ignite/table/TupleImpl.java

```diff
@@ -358,6 +360,17 @@ private void readObject(java.io.ObjectInputStream in) throws IOException, ClassN
         }
     }
 
+    /**
+     * Serializes an object. Required to be implemented in pair with the {@link #readObject(ObjectInputStream)} so that the
```

Comment:
Should we fix our marshalling code to comply to the Java Serialization spec instead? Maybe this should not be done in this PR, but then please file an issue and leave a TODO here